patches/third_party-libaddressinput-chromium-chrome_metadata_source.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/libaddressinput/chromium/chrome_metadata_source.cc b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
+index ffd583eea8eb..bcda8a02dc19 100644
+--- a/third_party/libaddressinput/chromium/chrome_metadata_source.cc
++++ b/third_party/libaddressinput/chromium/chrome_metadata_source.cc
+@@ -57,7 +57,7 @@ ChromeMetadataSource::Request::Request(
+ void ChromeMetadataSource::Download(const std::string& key,
+                                     const Callback& downloaded) {
```

Comment:
Could we just add a return; here instead of the modification below?

Reply:
No, because the caller expects that the `downloaded` callback will always be called, even if the download fails.

Reply:
got it thx